components/dashboard/src/settings/SelectIDE.tsx

```diff
@@ -176,42 +176,40 @@ function renderIdeOption(
     version: IDEOption["imageVersion"],
     onSelect: () => void,
 ): JSX.Element {
-    const label = option.type === "desktop" ? "" : option.type;
+    const shouldShowOptionType = option.type !== "desktop" || option.title === "VS Code"; // Force show of "Desktop" in the list for VS Code Desktop
     const card = (
         <SelectableCardSolid className="w-36 h-44" title={option.title} selected={selected} onClick={onSelect}>
-            <div className="flex justify-center mt-3">
-                <img className="w-16 filter-grayscale self-center" src={option.logo} alt="logo" />
-            </div>
-            <div
-                className="mt-2 px-3 py-1 self-center"
-                style={{
-                    minHeight: "1.75rem",
-                }}
-            >
-                {label ? (
-                    <span
-                        className={`font-semibold text-sm ${
-                            selected ? "text-gray-100 dark:text-gray-600" : "text-gray-600 dark:text-gray-500"
-                        } uppercase`}
-                    >
-                        {label}
-                    </span>
-                ) : (
-                    <></>
-                )}
-            </div>
-
             {version ? (
-                <div
-                    className={`font-semibold text-xs ${
-                        selected ? "text-gray-100 dark:text-gray-600" : "text-gray-600 dark:text-gray-500"
-                    } uppercase px-3 self-center`}
+                <span
```

Comment:
**tip**: we're using the web fonts that are available to user's system for now:

https://github.com/gitpod-io/gitpod/blob/4aacec9e264892232d76aba227f02e6f33da13f5/components/dashboard/tailwind.config.js#L50-L68